### DISCUSSION TITLE
i forgor 💀

### DISCUSSION THREAD

**Discussion by redstone-dev** [jeffalo/ocular#156] [Q&A] [answered]
i forgor 💀 how to display post count

**Comment by redstone-dev** [comment, marked as answer]
i rember

**Comment by Weredime** [comment]
{count} - see https://ocular.jeffalo.net/docs

**Reply by redstone-dev** [reply]
Thanks but I know now, I went digging through the code lol

Edit: thanks for the docs link, it'll help in the future lol
